### PR TITLE
v0.11.1 dev

### PR DESCRIPTION
**Pull request by losh11**
This is a sync as a continuation of the effort to build a litecoin fork of lnd v0.9.0. Although btcwallet's latest commit is not tagged, we choose to tag this v0.11.1.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,19 +1,35 @@
...
+	github.com/kkdai/bstream v0.0.0-20181106074824-b3251f7901ec // indirect
+	github.com/lightninglabs/gozmq v0.0.0-20191113021534-d20a764486bf
+	github.com/lightninglabs/neutrino v0.11.0
+	golang.org/x/crypto v0.0.0-20190211182817-74369b46fc67
+	golang.org/x/net v0.0.0-20190206173232-65e2d4e15006
+	google.golang.org/genproto v0.0.0-20190201180003-4b09977fb922 // indirect
 	google.golang.org/grpc v1.18.0
 )
 
+replace github.com/btcsuite/btcwallet/walletdb => ./walletdb
```
Comment (ghost) [inline go.mod]: need to fix the paths in the 5 "replace" lines here (btcsuite => ltcsuite, btcwallet => ltcwallet)

File: go.mod [hunk 2]
```diff
@@ -1,19 +1,35 @@
...
 	github.com/golang/protobuf v1.2.0
 	github.com/jessevdk/go-flags v1.4.0
 	github.com/jrick/logrotate v1.0.0
-	github.com/lightninglabs/gozmq v0.0.0-20180324010646-462a8a753885
-	go.etcd.io/bbolt v1.3.3 // indirect
-	golang.org/x/crypto v0.0.0-20190701094942-4def268fd1a4
-	golang.org/x/net v0.0.0-20190404232315-eb5bcb51f2a3
+	github.com/kkdai/bstream v0.0.0-20181106074824-b3251f7901ec // indirect
+	github.com/lightninglabs/gozmq v0.0.0-20191113021534-d20a764486bf
+	github.com/lightninglabs/neutrino v0.11.0
```
Comment (ghost) [inline go.mod]: we'll be using the ltcsuite/neutrino module here

File: chain/neutrino.go [hunk 3]
```diff
@@ -19,7 +18,7 @@ import (
 	"github.com/ltcsuite/neutrino"
```
Comment (ghost) [inline chain/neutrino.go:19]: add dependency	"github.com/ltcsuite/neutrino/headerfs"

File: chain/neutrino.go [hunk 4]
```diff
@@ -400,7 +399,7 @@ func (s *NeutrinoClient) Rescan(startHash *chainhash.Hash, addrs []ltcutil.Addre
 	}
 
 	s.clientMtx.Lock()
-/* commented out until ltcsuite/neutrino is updated
+	/* commented out until ltcsuite/neutrino is updated
```
Comment (ghost) [inline chain/neutrino.go]: this section was commented out last time we updated, due to un-solved dependency issues.  We should un-comment it out now, to make it match btcsuite.

File: chain/neutrino.go [hunk 5]
```diff
@@ -457,7 +456,7 @@ func (s *NeutrinoClient) NotifyReceived(addrs []ltcutil.Address) error {
 	s.lastFilteredBlockHeader = nil
 
 	// Rescan with just the specified addresses.
-/* commented out until ltcsuite/neutrino is updated
+	/* commented out until ltcsuite/neutrino is updated
```
Comment (ghost) [inline chain/neutrino.go]: this section was commented out last time we updated, due to un-solved dependency issues. We should un-comment it out now, to make it match btcsuite.

File: go.mod (hunk 1 above)
Comment (losh11) [reply go.mod]: fixed by https://github.com/ltcsuite/ltcwallet/pull/4/commits/5d8548ab35e2ba7a0b06a31f1a1b6a6e5937a081

File: go.mod (hunk 2 above)
Comment (losh11) [reply go.mod]: fixed with https://github.com/ltcsuite/ltcwallet/pull/4/commits/91b5386886079db1dbebac1333d4b9ccde6b21ab

File: chain/neutrino.go (hunk 4 above)
Comment (losh11) [reply chain/neutrino.go]: I think PR #5 fixes this. Can you review that please?

**Review by ghost — APPROVED** [review]

**Review by ghost — APPROVED** [review]

File: chain/neutrino.go (hunk 4 above)
Comment (ghost) [reply chain/neutrino.go]: done  :+1:

**Review by michael1011 — APPROVED** [review]

**Pull request merged**